scripts/encapp_quality.py

```diff
@@ -491,6 +493,16 @@ def run_quality(test_file, options, debug):
             failed["error"] = "Not an encapp file"
             return failed
 
+        # check if the encoder is lcevc
```

Comment:
Is it not possible to deduce tis from ffmpeg parsing?
It would be better if it was handled at lower level.